modules/@apostrophecms/attachment/index.js

```diff
@@ -467,7 +467,9 @@ module.exports = {
         }
 
         if (!self.croppable[info.extension]) {
-          throw new Error(info.extension + ' files cannot be cropped, do not present cropping UI for this type');
+          throw self.apos.error('invalid', req.t('apostrophe:fileTypeCannotBeCropped', {
```

Comment:
You'll never see this (it's a sanity check for a situation the UI should avoid) but I wanted to format it properly anyway. I was seeing it until I fixed autocrop to avoid it.